docs/releases/1.10.1.txt

```diff
@@ -56,3 +56,5 @@ Bugfixes
 * Reallowed the ``{% for %}`` tag to unpack any iterable (:ticket:`27058`).
 
 * Fixed ``makemigrations`` crash if a database is read-only (:ticket:`27054`).
+
+* Fixed duplicated managers in `Model._meta.managers` (:ticket:`27073`).
```

Comment:
how about, "Fixed duplicated managers when overriding a parent model's manager"
(would need double backticks if you keep Model._meta...)